static/css/dropdown-menu.css

```diff
@@ -0,0 +1,5 @@
+.dropdown-menu {
+    background: black;
+    border-radius: 10px;
+    box-shadow: 0px 0px 9px 2px rgb(73, 73, 73);
```

Comment:
Can you lighten the shadow a bit? Right now, it looks quite a lot raise in light theme on white background